cmd/largeUpload.go

```diff
@@ -0,0 +1,35 @@
+//go:build wip
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/storacha/guppy/cmd/internal/largeupload"
+)
+
+var largeUploadCmd = &cobra.Command{
+	Use:   "large-upload",
+	Short: "WIP - Upload a large amount of data to the service",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		space, err := cmd.Flags().GetString("space")
+		if err != nil {
+			return err
+		}
+
+		resumeUpload, err := cmd.Flags().GetBool("resume")
+		if err != nil {
+			return err
+		}
+
+		root := cmd.Flags().Arg(0)
+
+		return largeupload.Action(cmd.Context(), space, resumeUpload, root)
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(largeUploadCmd)
+
+	largeUploadCmd.Flags().String("space", "", "DID of the space to upload to")
+	largeUploadCmd.Flags().Bool("resume", false, "Resume a previously interrupted upload")
```

Comment:
`StringVar`, `BoolVar`, `<Type>Var` might be nice here for simpler flag parsing.